c2corg_ui/static/js/imageuploader.js

```diff
@@ -477,7 +477,9 @@ app.ImageUploaderModalController.prototype.close = function() {
  * @export
  */
 app.ImageUploaderModalController.prototype.save = function() {
+  $('.img-upl-button').addClass('disabled');
```

Comment:
More Angular-like would be to use [ng-class](https://docs.angularjs.org/api/ng/directive/ngClass) and to store the disabled status in a variable on the controller.